charts/airbyte/values.yaml

```diff
@@ -1319,10 +1325,10 @@ minio:
     tag: RELEASE.2023-11-20T22-40-07Z
 
   auth:
-    rootUser: minio
-    rootPassword: minio123
+    rootUser: ""
```

Comment:
What happens when you remove these?

Reply:
I don't believe it is being used anywhere other than `secret.yaml`, but I kept it here just in case removing this breaks anything because of the way that the Helm chart is defined.